### ISSUE TITLE
Crimea is shown as part of Russia

### ISSUE THREAD

**Issue by systemcatch** [tmrowco/electricitymap#1122]
This is due to the natural earth shapefiles assigning administration of the area to Russia. We can alter this by using the state file instead and assigning Crimea to Ukraine. Lets discuss what we want to do here.

**Comment by corradio** [comment]
Any statement by the UN about this? We should probably follow the position of the UN on this.

**Comment by systemcatch** [comment]
They support Ukraine.
https://en.m.wikipedia.org/wiki/United_Nations_General_
Assembly_Resolution_68/262

**Comment by jarek** [comment]
Do we know if UKRENERGO generation figures still include generation in Crimea? It looks like Crimea has about 500 MW of generation capacity and SO-UPS currently shows 757 MW of exports to Crimea (distinct from their report on interconnection to Ukraine). Perhaps we should have Crimea as a separate region altogether.

**Comment by systemcatch** [comment]
https://sputniknews.com/russia/201601141033156976-crimea-switches-away-from-ukrainian-power/ suggests that Ukraine no longer is no longer supplying Crimea and that Russia is providing power.

**Comment by systemcatch** [comment]
Maybe the simplest way to solve this is to create a new zone UKR-KR using data from http://br.so-ups.ru/Public/MainPageData/VSVGO/PowerVIEByRegions.aspx. Assuming that page updates scheduled to actual similar to the RU parser.

**Comment by corradio** [comment]
If we can confirm that Crimea is not part of the RU parser, then let's create a new zone, and consider it as a "region" of Ukraine.

Maybe we should use the following codes?
https://en.wikipedia.org/wiki/ISO_3166-2:UA

**Comment by systemcatch** [comment]
Yup those codes look good. I think that Crimea's generation is included in the RU parser at the moment which makes things difficult, @alixunderplatz do you think the generation schedule on http://br.so-ups.ru/Public/MainPageData/VSVGO/PowerVIEByRegions.aspx is updating once the hour has passed like in the main RU parser? I've been watching it but haven't seen anything yet.

**Comment by alixunderplatz** [comment]
@systemcatch it should be included in the RU parser, that is correct, because it is part of price zone 1.

I just saw I gave the wrong link for thermal generation above - excuse me! The link in your comment is for the generation forecast for renewable energies (in this case, solar+wind), which probably remains constant through the day.
Here is the link for thermal generation (I fixed it now in my longer comment above):
http://br.so-ups.ru/Public/MainPageData/VSVGO/ActivePowerTPPByRegions.aspx
Lets check if that one changes a little through time.
